state/execution.go

```diff
@@ -311,15 +311,15 @@ func (blockExec *BlockExecutor) CheckBlock(block *types.Block) error {
 		DeliverBlock: &abcix.ResponseDeliverBlock{Events: resp.Events, DeliverTxs: resp.DeliverTxs}})
 	if !bytes.Equal(resultHash, block.Header.ResultsHash.Bytes()) {
 		blockExec.logger.Error(
-			"resultHash mismatch.", "ResultHash in ResponseCheckBlock:", resultHash,
+			"resultHash mismatch.", "ResultHash in ResponseCheckBlock", resultHash,
 			"ResultHash in block header:", block.Header.ResultsHash.Bytes(),
```

Comment:
same with colon here. 
I would suggest using one word instead of a phrase as the key, or `blockExecLogger.Error(fmt.Sprintf(...))` is also a good way to wrap the err msg.